modules/dnn/src/layers/layers_common.cpp

```diff
@@ -172,6 +170,7 @@ void getPoolingKernelParams(const LayerParams &params, std::vector<size_t>& kern
     {
         util::getKernelSize(params, kernel);
     }
+    util::getStrideAndPadding(params, pads_begin, pads_end, strides, padMode, !globalPooling? kernel.size() : 2);
```

Comment:
I think we can keep it unchanged, can you check?

Reply:
No, we need to know kernel.size to set correct pads and strides size